macros/models/_sector/tokens/transfers_enrich.sql

```diff
@@ -14,6 +14,7 @@ END
     , evms_info_model = null
     , transfers_start_date = '2000-01-01'
     , blockchain = null
+    , usd_amount_threshold = 1000000000 -- USD amount threshold
```

Comment:
this is currently set to 1 billion. options to set threshold and their count of rows filtered out:
- 1 billion: 530 rows
- 10 billion: 58 rows
- 100 billion: 43 rows

Reply:
not sure what the magic number should be. i'm following up with the team internally.

Reply:
```suggestion
    , usd_amount_threshold = 25000000000 -- USD amount threshold
```

Reply:
the team landed on $25b